src/synth-kitchen/kitchen.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import { Patch } from './ui/patch';
+
+export const Kitchen: React.FunctionComponent = () => {
+	return (
+		<Patch />
```

Comment:
This is kind of the point of this PR. I want the Patch to be a self-contained component. Eventually we can add an API to give it a serialized state as a starting point.